riscv/src/continuations/bootloader.rs

```diff
@@ -40,15 +40,13 @@ const_assert!(PAGE_SIZE_BYTES > 384);
 
 /// Computes the size of the bootloader given the number of input pages.
 pub fn bootloader_size(accessed_pages: &BTreeSet<u32>) -> usize {
-    let constant_overhead = 1 + // jump bootloader_init
-        2 + // load number of pages
+    let constant_overhead = 2 + // load number of pages
         8 + // init memory hash
         1 + // page idx = 0
         1 + // branch_if_diff_equal if no pages
         8 + // assert final merkle root
         REGISTER_MEMORY_NAMES.len() + // load memory regs
-        REGISTER_NAMES.len() * 2 +  // load asm regs
-        1; // jump_to_bootloader_input
+        REGISTER_NAMES.len() * 2;
```

Comment:
```suggestion
        REGISTER_NAMES.len() * 2;  // load asm regs
```

This is still relevant?